### ISSUE TITLE
Retrieve the list of Actions of a Viz

### ISSUE THREAD

**Issue by florent-guinard** [tableau/embedding-api-v3-guide#23]
We'd like to be able to retrieve the list of Actions (Worksheet/Dashboard > Actions in **Tableau**) for a Worksheet/Dashboard in an embedded Workbook:
![image](https://user-images.githubusercontent.com/93193181/156206758-0b45a164-2a55-41bf-ac5d-50571a3bf5f5.png)

We're looking for as much information as possible on those actions, like:
- Type
- Name
- Run action on
- Target
- "Clearing the selection will"
- Fields
- URL for URL Actions

Thanks !

**Comment by illonage** [comment]
Hi @florent-guinard - Thank you for opening this issue. Do you mind sharing more about your use case please?

**Comment by Mrkbingham** [comment]
Wow this is a great request, would love to get access to this info!

**Comment by illonage** [comment]
Thanks @florent-guinard ! @Mrkbingham do you have the same use cases?

**Comment by Mrkbingham** [comment]
@illonage yep!  Somewhat similar, I’d like to be able to read all of the settings that someone has set up inside Tableau so my app can detect the set behavior.  The major config that would be helpful are: Target, Fields and URL (also the href target, e.g. open in new tab)

The URL would help ensure that we’re opening a URL that may already be embedded in our app, if not we send the users over to Tableau Server.

The fields would be helpful for tracking user behavior - we currently track filter events in the applyFilter callback, and would like to continue that.  

Lastly the name of the action: we log errors for unexpected behavior, and if we could say “Action [Title] failed” it would help immensely for users when troubleshooting.
